### DISCUSSION TITLE
how can i install specific version of code-server

### DISCUSSION THREAD

**Discussion by hakanaku1234** [coder/code-server#3099] [Q&A] [answered]
hi, with `install.sh`, how can i specific the version of code-server to install?

since the latest version has some bugs, i want to downgrade it.
such as [1] when i run `code-server foo.py`, it open a new window session instead of just open a new file and attach to editor group.
and [2] sometimes the shortcut ctrl+` didn't toggle (open) the terminal when focused on editor.

**Comment by bpmct** [comment, marked as answer]
You can specify this with the --version flag :)

reference: https://github.com/cdr/code-server/blob/main/install.sh#L155

You can also do `install.sh --help` to see all of the options :)

**Reply by hakanaku1234** [reply]
ok thanks :) !!

**Comment by bpmct** [comment]
also for [1], I believe you need to specify the -r flag to "reuse" the existing window. This is just a new feature of VS Code I believe.

**Reply by bpmct** [reply]
Also, if you are still experiencing #2, could you create a [new issue](https://github.com/cdr/code-server/issues/new/choose) for this? We'd love to fix it :)
